django_tasks/task.py

```diff
@@ -24,6 +24,7 @@
     from .backends.base import BaseTaskBackend
 
 DEFAULT_QUEUE_NAME = "default"
+DEFAULT_TASK_BACKEND_ALIAS = "default"
```

Comment:
I am really not sure about the location of that constant. The locations where I wanted to put it produced circular imports, though...

Reply:
I think this is why I just duplicated it instead. Tests should cover if they're different.